sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -1327,3 +1327,30 @@ def test_session_id_str_bytes(self, uamqp_transport, *, servicebus_namespace_con
                     messages = receiver.receive_messages(max_wait_time=10)
                     assert len(messages) == 1
                     assert messages[0].session_id == session_id
+    
+    
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', requires_session=True)
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasser()  
+    def test_next_available_session_timeout_value(self, uamqp_transport, *, servicebus_namespace_connection_string=None, servicebus_queue=None, **kwargs):
+        if uamqp_transport==True:
+            pytest.skip("This test is for pyamqp only")
+        with ServiceBusClient.from_connection_string(
+            servicebus_namespace_connection_string, logging_enable=False, uamqp_transport=uamqp_transport, retry_total=1) as sb_client:
+            
+            receiver = sb_client.get_queue_receiver(servicebus_queue.name, session_id=NEXT_AVAILABLE_SESSION, max_wait_time=10,)
```

Comment:
what happens if you have an idle_timeout value >/</= max_wait_time?
may want to test this

Reply:
This would play in to a non session queue as well.  Where it idle timesout waiting for the first message. I would expect greater value to win.

Reply:
ServiceBus doesn't have idle_timeout, that is an Event Hubs kwarg - we have socket_timeout